src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final double DEFAULT_RADIUS = 10.0;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public BaseShape getRandomFigure() {
```

Comment:
Don't begin method implementation with an empty line. Please remove the redundant empty line.